lib/acme/crypto.rb

```diff
@@ -13,14 +13,16 @@ def generate_signed_jws(header:, payload:)
     jwt.to_json(syntax: :flattened)
   end
 
-  def jwk
-    @jwk ||= JSON::JWK.new(public_key)
-  end
-
   def thumbprint
     jwk.thumbprint
   end
 
+  private
```

Comment:
I guess we miss a spec calling this https://github.com/unixcharles/acme-client/blob/master/lib/acme/resources/challenges/dns01.rb#L14